combat-trainer.lic

```diff
@@ -557,6 +578,26 @@ class LootProcess
     echo "Last ritual performed: #{@last_ritual}" if $debug_mode_ct
   end
 
+  def butcher_corpse(mob_noun, ritual, game_state)
+    return unless ritual.eql?('butcher')
+    echo "Butchering the #{mob_noun}'s corpse!" if $debug_mode_ct
+    echo " Only butchering it once!" if $debug_mode_ct && butcher_once
+
+    bput("perform preserve on #{mob_noun}", @rituals['preserve'], @rituals['failures'])
```

Comment:
Should this be `do_necro_ritual(mob_noun, 'preserve', game_state)`?

Reply:
Yes, I'll add that, duh.

Reply:
Just make sure you retest it, `do_necro_ritual` does alter some internal state.